sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -14,13 +14,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 """SDK Fn Harness entry point."""
 
+import BaseHTTPServer
```

Comment:
The changes to this file is reviewed in a different PR. Could you revert this here? Are there any parts that needs to be reviewed in this PR?

Reply:
I think it spilled out into this PR while rebaseing on other PR.
I will fix it.